src/pages/index.js

```diff
@@ -66,20 +111,87 @@ function renderCard(data) {
 /*-------------------------------------------------------------------------------*/
 
 function handleProfileFormSubmit(data) {
-  userInfo.setUserInfo(data);
-  profileFormValidator.resetValidation();
-  profileEditPopup.close();
+  profileEditPopup.renderLoading(true);
+  api
+    .updateUserInfo(userInfo.getUserInfo())
```

Comment:
`    .updateUserInfo(userInfo.getUserInfo())
`
* When I reload the page, the old profile data appears again

`data` is needed here

` .updateUserInfo(data)`